source/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -58,10 +58,9 @@ todosController.get('remaining');
 Note that `@each` only works one level deep. You cannot use nested forms like
 `todos.@each.owner.name` or `todos.@each.owner.@each.name`.
 
-Sometimes, you may only care if the items of an array have been added, removed, or replaced, 
-rather than changes to properties on the items themselves. For example, items may have been 
-added to or removed from the array. In this case, you can create a computed property as follows:
-
+Sometimes you care about changes to the contents of the array (i.e. items are added or
+removed, or the property is set to a new array), but not changes to properties of
```

Comment:
> or the property is set to a new array

This is explicitly *not* a change to the contents of an array. It is replacing a property with a new value.